### DISCUSSION TITLE
How to logout from bot login?

### DISCUSSION THREAD

**Discussion by cloudskylimit** [aahnik/tgcf#32] [Q&A]
I run forwarder.py, and I accidentally input my bot token.
now I can't run the forwarder.py anymore, because I got some errors like these:

```shell
telethon.errors.rpcerrorlist.BotMethodInvalidError: The API access for bot users is restricted. The method you tried to invoke cannot be executed as a bot (caused by SearchRequest)
```

**Comment by aahnik** [comment]
@cloudskylimit  Simply delete the `.session` files.

This script can be run using bot accounts too, but I will add documentation for that later.

**Reply by cloudskylimit** [reply]
Thank you, that's good to hear.
I will wait for the documentation then 👍
